src/data/repack.cc

```diff
@@ -15,20 +15,16 @@
  *
  */
 #include "data/repack.h"
+
 #include <stdlib.h>
 
+#include "util/math.h"
```

Comment:
Mhm, then never mind - I was confused by imports from our project both on top and bottom of list, but it seems to be in line with those guidelines.